visit-scheduler-api-specification.yaml

```diff
@@ -429,13 +429,13 @@ components:
         dateOfBirth:
           type: string
           description: prisoner date of birth
-    Establishment:
+    prison:
       type: object
-      description: Establishment Description
+      description: prison Description
       properties:
-        establishmentId:
+        prisonId:
           type: string
-          description: Unique identifier for this establishment
-        establishmentName:
+          description: Unique identifier for this prison
+        prisonName:
           type: string
```

Comment:
Discussion point.  The front end service will be looking up descriptions and reference data - should prison name lookup be done there too?

Reply:
Looking at the response of other API endpoints, they seem to always return both the code and the description - so I followed suit - it doesn't seem worth the effort of the extra call to get the name.